ui/raidboss/timeline.ts

```diff
@@ -150,7 +151,14 @@ export class Timeline {
   }
 
   private LoadFile(text: string, triggers: LooseTimelineTrigger[], styles: TimelineStyle[]): void {
-    const parsed = new TimelineParser(text, this.replacements, triggers, styles, this.options);
+    const parsed = new TimelineParser(
+      text,
+      this.replacements,
+      triggers,
+      styles,
+      this.options,
+      this.zoneId,
```

Comment:
Very nit, if you want: weird to have zoneId be last here when it comes before text everywhere else?

Reply:
Mmm in this case it's because it's optional but text is not optional.  It's also not optional in the other paths.  Do you think I should put it last elsewhere?

Reply:
No, I feel very not strongly, just something I saw

Reply:
Done, all the same! :+1: